ui/component/settingNotifications/view.jsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import Card from 'component/common/card';
+import { FormField } from 'component/common/form';
+import SettingsRow from 'component/settingsRow';
+import ChannelSelector from 'component/channelSelector';
+
+type Props = {
+  // --- select ---
+  notificationSettings: ?NotificationSettings,
+  // --- perform ---
+  doFetchNotificationSettings: () => void,
+  doToggleNotificationsDisabled: () => void,
+  doToggleFollowerNotifications: () => void,
+  doToggleFollowedNotifications: () => void,
+};
+
+export default function SettingNotifications(props: Props) {
+  const {
+    notificationSettings,
+    doFetchNotificationSettings,
+    doToggleNotificationsDisabled,
+    doToggleFollowerNotifications,
+    doToggleFollowedNotifications,
+  } = props;
+
+  const { disabled, mention } = notificationSettings || {};
+  const { all: notificationsDisabled } = disabled || {};
+  const { from_followed: followedMentionsEnabled, from_followers: followerMentionsEnabled } = mention || {};
+
+  React.useEffect(() => {
+    if (doFetchNotificationSettings && !notificationSettings) doFetchNotificationSettings();
+  }, [doFetchNotificationSettings, notificationSettings]);
+
+  return (
+    <>
+      <ChannelSelector hideAnon />
+
+      <Card
+        isBodyList
+        body={
+          <>
+            <SettingsRow title={__('Disable All')}>
```

Comment:
Maybe a subtitle to explain `Disable All` (what is it disabling ... is it graying out Mention Notifications, or including Email and Browser notifications, since these are in the same page) and `Mention Notification`.